plugins/permission-node/api-report.md

```diff
@@ -115,17 +115,30 @@ export const createConditionTransformer: <
 export const createPermissionIntegrationRouter: <
   TResourceType extends string,
   TResource,
->(options: {
+>(
+  options: CreatePermissionIntegrationRouterOptions<TResourceType, TResource>,
+) => express.Router;
+
+// @public
+export type CreatePermissionIntegrationRouterOptions<
```

Comment:
What you could also do is overload the function, that can display a bit nicer when you have a set of separate of different options:

```ts
function createPermissionIntegrationRouter(options: { permissions: Array<Permission> }): Router;
function createPermissionIntegrationRouter<TResourceType extends string, TResource>(options: CreatePermissionIntegrationRouterResourceOptions<TResourceType, TResource>): Router;
function createPermissionIntegrationRouter<TResourceType extends string, TResource>(options: { permissions: Array<Permission> } | CreatePermissionIntegrationRouterResourceOptions<TResourceType, TResource>)) {
  ...
}
```